### PR TITLE
Clarify namespace update result with --verbose flag

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

- Started printing Namepeace Update request as JSON
- Rewritten namespace update tests as E2E tests instead of unit
- Increased context deadline for workflow update tests

## Why?

- Help users when running into this issue, resolve https://github.com/temporalio/cli/issues/209
- Workflow update tests would occasionally fail due to context deadline errors

## Checklist

1. Closes 

2. How was this tested:

Added E2E tests

3. Any docs updates needed?

### CODE REVIEW

**Review by bergundy — COMMENTED** [review]
Let's just print out what changed instead of diffing.
I don't see a good enough reason to do the extra describe call.

**Comment by feedmeapples** [comment]
> Let's just print out what changed instead of diffing. I don't see a good enough reason to do the extra describe call.

I started with this initially, seemed harder to maintain as ended up with `if` case for each property and filling in a map of change

**Comment by bergundy** [comment]
I mean just print out what we sent to the server, not the actual diff.

File: app/namespace_test.go [hunk 1]
```diff
@@ -70,53 +70,6 @@ var describeNamespaceResponseServer = &workflowservice.DescribeNamespaceResponse
...
-	resp := describeNamespaceResponseServer
-	s.frontendClient.EXPECT().DescribeNamespace(gomock.Any(), gomock.Any()).Return(resp, nil).Times(2)
-	s.frontendClient.EXPECT().UpdateNamespace(gomock.Any(), gomock.Any()).Return(nil, nil).Times(2)
-	err := s.app.Run([]string{"", "operator", "namespace", "update", cliTestNamespace})
-	s.Nil(err)
-	err = s.app.Run([]string{"", "operator", "namespace", "update", "--description", "another desc", "--email", "another@uber.com", "--retention", "1", cliTestNamespace})
-	s.Nil(err)
-}
-
-func (s *cliAppSuite) TestNamespaceUpdate_Data() {
```
Comment (feedmeapples) [inline app/namespace_test.go:83]: Re-created these as E2E tests. Removed unit tests as these do not really test much, even here the usage was wrong and it read `"true"` as the name of the namespace

**Review by samanbarghi — APPROVED** [review]

**Pull request merged**